release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -57,6 +57,12 @@ This release adds improvements related to the following components and concepts.
 [id="ocp-4-14-installation-and-update"]
 === Installation and update
 
+[id="ocp-4-14-nat-gateway-azure"]
+==== Installing a cluster on Microsoft Azure using a NAT gateway (Technology Preview)
+
+In {product-title} {product-version}, you can install a cluster that uses a NAT gateway for outbound networking. This is available as a Technology Preview (TP). For more information, see xref:../installing/installing_azure/installation-config-parameters-azure.adoc#installation-configuration-parameters-additional-azure_installing-azure-customizations[Additional Azure configuration parameters].
```

Comment:
Did you want the xref to go directly to the "Additional Azure configuration parameters" section? If so I think the xref needs to be `../installing/installing_azure/installation-config-parameters-azure#installation-configuration-parameters-additional-azure_installation-config-parameters-azure`

Reply:
You're quite right, thanks